packages/grafana-ui/src/components/Table/TableNG/Cells/DataLinksCell.tsx

```diff
@@ -0,0 +1,50 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+import { useStyles2 } from '../../../../themes';
+import { CellNGProps } from '../types';
+import { getCellLinks } from '../utils';
+
+export const DataLinksCell = (props: CellNGProps) => {
+  const { field, rowIdx } = props;
+  const styles = useStyles2(getStyles);
+
+  const links = getCellLinks(field, rowIdx!);
+
+  return (
+    <div>
+      {links &&
+        links.map((link, idx) => {
+          return (
+            // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+            <span key={idx} className={styles.linkCell} onClick={link.onClick}>
+              <a href={link.href} target={link.target}>
+                {link.title}
+              </a>
+            </span>
+          );
+        })}
+    </div>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => ({
```

Comment:
Do we need to add support for `cellLinkForColoredCell` here?

Reply:
No, if we want parity with old table. I think only `AutoCell` needs it.